README.md

```diff
@@ -1,18 +1,22 @@
 # USRSE Logos
 
 This is a repository of logos, presentation templates, and other associated
-branding files for the USRSE oranization.
+branding files for the USRSE organization. Here is a small preview (200x255)
+of the logo:
 
+![png/rse-logo-200x255.png](png/rse-logo-200x255.png)
```

Comment:
Hey @vsoch am I just blind, or does this file (png/rse-logo-200x255.png) not exist?

Reply:
That's weird - I thought these were in the the repo at some point... I have them on my filesystem! I'm not comfortable with the amount of changes that are going into this repository without any kind of PR, so I'm going to step off from working / maintaining it. Someone else can do this change (required by @danielskatz).